apps/prairielearn/src/lib/editors.ts

```diff
@@ -731,7 +759,18 @@ export class AssessmentAddEditor extends Editor {
     const oldNamesShort = await this.getExistingShortNames(assessmentsPath, 'infoAssessment.json');
 
     debug('Generate TID and Title');
-    const names = getNamesForAdd(oldNamesShort, oldNamesLong);
+    const names = getNamesForAdd(
+      oldNamesShort,
+      oldNamesLong,
+      this.aid,
+      this.title,
+      true, // This is enabled to handle duplicate short names case-insensitively. A duplicate in a different case
+      // results in a directory conflict.
+
+      // e.x. if the user tries to add an assessment with the short name "Test" when an assessment
```

Comment:
This comment should be moved to the function definition for comparing names in my opinion.